bin/insert

```diff
@@ -74,7 +76,7 @@ function main() {
     "${BASH_SOURCE%/*}/authenticate"
 
     prefix=${BUCKET_PREFIX}/${PUBLIC_KEY_HEX_EXTERNAL}/${APP_NAME}/${SUBMISSION_DATE}
-    gsutil -m cp -r "${BUCKET_INTERNAL_PRIVATE}/${prefix}/processed/publish" "${data_in}"
+    gsutil -m cp -r "gs://${BUCKET_INTERNAL_PRIVATE}/${prefix}/processed/publish" "${data_in}"
```

Comment:
Buckets are unprefixed again, because of the way that minio handles different endpoints.